### PR TITLE
closes #60 feature show files in dropdown

### LINKED ISSUE #60: [Feature]: Show files in dropdown
**Is your feature request related to a problem? Please describe.**
The dropdown is now using static test inputs to show its content instead of showing the uploaded metrics and sites.

**Describe the solution you'd like**
That the uploaded files should be sent to the dropdown constructor so they can be accurately be displayed. Also this should be updated when the uploaded files change.

**Additional context**
#N/A

**Depends on completion of issues:**
#22 #8 #9

### PR DESCRIPTION
**Pull request by rajtarn**

### CODE REVIEW

File: src/cpp/file_handler.cpp [hunk 1]
```diff
@@ -290,3 +296,45 @@ void FileHandler::AddPerformanceFile(std::string &file, std::string &file_name)
 std::string FileHandler::GetIdFromPerformance(std::string &file_name) const {
     return  file_name.substr(0, file_name.find("_"));
 }
+
+std::string FileHandler::GetSiteNames() const {
+    json sitesNames;
+    std::vector<std::string> names;
+    sitesNames["names"] = names;
+
+    for (auto i : sites) {
```
Comment (Dethcrvsh) [inline src/cpp/file_handler.cpp:305]: Since this is an object it should probably not be named 'i'

File: src/cpp/file_handler.cpp [hunk 2]
```diff
@@ -290,3 +296,45 @@ void FileHandler::AddPerformanceFile(std::string &file, std::string &file_name)
...
+    json metric_json;
+    std::vector<json> metrics;
+    metric_json["metrics"] = metrics;
+
+    for (auto i : sites) {
+        json categories;
+        CalculateCategories(i.second, categories);
+        // add all metric names
+        for (auto &el : categories.items()) {
+            metric_json["metrics"].push_back(el.key());
```
Comment (Dethcrvsh) [inline src/cpp/file_handler.cpp:329]: This could use a set instead to avoid duplicates in the first place

**Review by Dethcrvsh — CHANGES_REQUESTED** [review]
See my comments

**Review by NilsArnlund — APPROVED** [review]
Nice work!

**Review by ELEMENTLHERO — APPROVED** [review]
Ger ok men skrev ett par kommentarer som man kanske vill kolla på

**Review by Dethcrvsh — APPROVED** [review]

**Pull request merged**